src/d3d12/d3d12_renderer.hpp

```diff
@@ -61,9 +63,11 @@ namespace wr
 			DirectX::XMMATRIX m_inverse_projection;
 			DirectX::XMMATRIX m_inv_vp;
 
-			uint32_t m_padding[2];
+			//uint32_t m_padding[0];
```

Comment:
Don't comment things out. We are using version control.

Reply:
removed